ng/src/web/pages/dfncert/detailspage.js

```diff
@@ -44,20 +47,35 @@ import ExternalLink from '../../components/link/externallink.js';
 
 import DfnCertAdvDetails from './details.js';
 
-const ToolBarIcons = () => (
-  <IconDivider>
-    <ManualIcon
-      page="vulnerabilitymanagement"
-      anchor="id15"
-      title={_('Help: DFN-CERT Advisories')}
-    />
-    <ListIcon
-      title={_('DFN-CERT Advisories')}
-      page="dfncertadvs"
+const ToolBarIcons = ({
+  entity,
+  onDfnCertAdvDownloadClick,
+}) => (
+  <Divider margin="10px">
+    <IconDivider>
+      <ManualIcon
+        page="vulnerabilitymanagement"
+        anchor="id15"
+        title={_('Help: DFN-CERT Advisories')}
+      />
+      <ListIcon
+        title={_('DFN-CERT Advisories')}
+        page="dfncertadvs"
+      />
+    </IconDivider>
+    <ExportIcon
```

Comment:
Once more ;) Just to make sure, nothing is forgotten in case the Dividers change.